packages/devtools_app/lib/src/screens/memory/memory_controller.dart

```diff
@@ -313,9 +260,12 @@ class MemoryController extends DisposableController
     );
   }
 
+  /// This flag will be needed for offline mode implementation.
+  bool offline = false;
+
```

Comment:
this actually will not be needed. We will use `offlineController.offlineMode.value` to check whether we are in offline mode or not.

Reply:
I want to preserve all references to this variable, so that we do not forget to review them and decide what will be their behavior in offline mode.